scripts/download/lima.mjs

```diff
@@ -7,7 +7,7 @@ import path from 'path';
 import { download, getResource } from '../lib/download.mjs';
 
 const limaRepo = 'https://github.com/rancher-sandbox/lima';
-const limaTag = 'v0.5.1';
+const limaTag = 'v0.5.2';
```

Comment:
@jandubois 

Next time could you consider using a version number like `v0.5.0-rancher.2`, to avoid having inconsistent versions across the upstream and Rancher?

Reply:
I think we need to have a `lima-and-qemu` repo that is not a fork of the `lima` repo, and that has its own versioning. That repo can then have `lima`, `vde_vmnet`, `qemu`, etc submodules to specify the exact version to build. We are going to need to build qemu from source anyways, at least for ARM, to apply the patches for the hypervisor framework support.

Not that it matter much, but `v0.5.0-rancher.2` would be somewhat confusing too, as per the semver rules, this is a pre-release of `v0.5.0`, and not something later.

Reply:
lima-and-qemu tarballs have now their own version number that is not linked to any component version: https://github.com/rancher-sandbox/lima-and-qemu/releases/tag/v1.0